patches/components-permissions-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
+index 2a4eb7e3a8c106687f80656051c6a0a9e70e0ce3..1e74754c75de0b7dd5e09f5bb27e8b4c98e5ab53 100644
+--- a/components/permissions/BUILD.gn
++++ b/components/permissions/BUILD.gn
+@@ -55,6 +55,7 @@ source_set("permissions") {
+     "//ui/base",
+     "//url",
+   ]
++  deps += [ "//brave/components/permissions" ]
```

Comment:
we should be adding a dep for `components_strings` in `components/strings/BUILD.gn` for this which will then be covered by `"//components/strings:components_strings_grit"` for all relevant targets

Reply:
Addressed by 3eac52f218f76979995a64f5bd536be932ef143f